### PR TITLE
Minor tweaks to the preface

### PR DESCRIPTION
**Pull request by jni**
This change is [Reviewable](https://reviewable.io/reviews/elegant-scipy/elegant-scipy/285)

### CODE REVIEW

File: markdown/preface.markdown [hunk 1]
```diff
@@ -164,9 +164,10 @@ scientific computing.
 
 In your Python travels, you may have already heard a few rumblings about which version of Python is better.
 You may have wondered why it's not just the latest version.
+(Spoiler alert: it is.)
 
 At the end of 2008, the Python core developers released Python 3.0 and, as
-Douglas Adams observed tongue-in-cheek about the creation of the Universe in
+Douglas Adams observed about the creation of the Universe in
```
Comment (stefanv) [inline markdown/preface.markdown]: We specifically added this in to help users unfamiliar with the joke understand what's happening.

File: markdown/preface.markdown [hunk 2]
```diff
@@ -243,8 +244,9 @@ unicode.  For example, `\beta<TAB>` becomes β.
 The Python 3 update also breaks a lot of existing 2.x code, and in some
 cases executes more slowly than before.  Despite these frustrations,
 we encourage all users to upgrade as soon as possible (Python 2.x is
-now in maintenance only mode until 2020), as we hope to see most
-issues addressed as the 3.x series matures.
+now in maintenance only mode until 2020), as most issues have been addressed as
```
Comment (stefanv) [inline markdown/preface.markdown]: "as ... as" reads uncomfortably

File: markdown/preface.markdown [hunk 3]
```diff
@@ -440,7 +442,8 @@ See the the book [GitHub repository](https://github.com/elegant-scipy/elegant-sc
 
 ### Accessing the book materials
 
-All of the code and data from this book is available on our GitHub repository: [https://github.com/elegant-scipy/elegant-scipy](https://github.com/elegant-scipy/elegant-scipy)
+All of the code and data shown in this book are available on our GitHub repository: [https://github.com/elegant-scipy/elegant-scipy](https://github.com/elegant-scipy/elegant-scipy).
+In the README file in that repository, you will find instructions to build Jupyter notebooks from the markdown source files, which you can then run interactively, using the data included in the repo.
```
Comment (stefanv) [inline markdown/preface.markdown]: Hrm, should we just upload those notebooks?  Asking users to get into conversion is too much.

File: markdown/preface.markdown (hunk 1 above)
Comment (jni) [inline markdown/preface.markdown]: @stefanv we added the whole sentence, but I think you can have `tongue-in-cheek` or `about the creation of the Universe`, but not both. It becomes both unwieldy and "explaining the obvious". I also made our stated position more obvious in the opening paragraph of the section.

File: markdown/preface.markdown (hunk 3 above)
Comment (jni) [inline markdown/preface.markdown]: The last time I suggested this, your response was to update the README with better instructions. =D Let's punt on this for now. I'll make a "repo cleanup" issue that includes this.

Comment (jni) [inline markdown/preface.markdown]: See #296

File: markdown/preface.markdown (hunk 1 above)
Comment (jni) [inline markdown/preface.markdown]: I finally realised that a synonym for "said tongue-in-cheek" is "quipped". =P

Also, we don't need the "in the hitchhiker's guide to the galaxy", that can be footnoted. Thoughts on present phrasing?

File: markdown/preface.markdown (hunk 3 above)
Comment (stefanv) [inline markdown/preface.markdown]: I thought you'd be happy when I come around to your POV :)

**Pull request merged**